frontend/src/app/admin/superadmin/structures/structures.component.scss

```
@@ -65,52 +65,124 @@
-    transition: all 0.2s ease;
-
-    &:hover {
-      transform: translateY(-2px);
-      box-shadow: 0 18px 32px rgba(15, 23, 42, 0.1);
-      border-color: #cbd5f5;
-    }
+.structure-tile {
+  border-radius: 18px;
+  border: 1px solid #e5e7eb;
+  background: linear-gradient(180deg, #ffffff 0%, #f8fafc 100%);
+  box-shadow: 0 14px 28px rgba(15, 23, 42, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)